### PR TITLE
Crossgen for linux-arm

### PR DESCRIPTION
**Pull request by eerhardt**
Generating crossgen'd images for linux-arm.

/cc @Petermarcu @RussKeldorph

### CODE REVIEW

File: netci.groovy [hunk 1]
```diff
@@ -59,10 +59,15 @@ platformList.each { platform ->
...
             }
             else {
-                dockerContainer = "ubuntu-14.04-cross-0cd4667-20172211042239"
+                dockerContainer = "ubuntu-14.04-cross-e435274-20180323032140"
             }
             dockerCommand = "docker run -e ROOTFS_DIR=/crossrootfs/${architecture} --name ${dockerContainer} --rm -v \${WORKSPACE}:${dockerWorkingDirectory} -w=${dockerWorkingDirectory} ${dockerRepository}:${dockerContainer}"
-            buildArgs += " -SkipTests=true -DisableCrossgen=true -CrossBuild=true"
+            buildArgs += " -SkipTests=true -CrossBuild=true"
+
+            if (architecture == 'armel' || architecture == 'arm64') {
```
Comment (janvorli) [inline netci.groovy]: Why do we disable it for armel? Just being curious.

Comment (eerhardt) [reply netci.groovy]: It was disabled before, so I wasn't changing that.

I'm not sure when and where we build for `linux-armel`.  The Tizen case doesn't get into this, so I kind of think that `armel` in this code is a dead code path.  We don't ever have os = "Linux" and architecture = "armel".

**Comment by eerhardt** [comment]
Fixes #3314

**Comment by eerhardt** [comment]
@dotnet-bot test ci please

**Comment by eerhardt** [comment]
CI run is here:  https://ci.dot.net/job/dotnet_core-setup/job/master/job/GenPRTest/job/release_linux_arm_prtest/2/console.  It passed.

**Pull request merged**